pyaedt/hfss.py

```diff
@@ -3085,7 +3085,7 @@ def create_wave_port_from_sheet(
                 faces = sheet
             else:
                 faces = self.modeler.get_object_faces(sheet)[0]
-            if not faces:
+            if not faces:  # pragma: no cover
                 self.logger.error("Wrong Input object. it has to be a face id or a sheet.")
                 return False
```

Comment:
This should be a `TypeError`.

Reply:
Agreed. It will be part of a global refactoring work.